### PR TITLE
Modern templates

### PR DESCRIPTION
**Pull request by sadrasabouri**
### Added
- `feature_request.yml` template
- `config.yml` for issue template

### Changed
- Bug report template modified

### CODE REVIEW

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
@sadrasabouri Thanks for your efforts 💯 
Please use `PyRGG` instead of `Pyrgg` in `.yml` files

**Comment by sadrasabouri** [comment]
Kindly please re-review @sepandhaghighi

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Discord
+    url: https://discord.com/invite/dfYAWVMaCW
+    about: Ask questions and discuss with other Pyrgg community members
```
Comment (sepandhaghighi) [inline .github/ISSUE_TEMPLATE/config.yml:5]: PyRGG

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (sadrasabouri) [reply .github/ISSUE_TEMPLATE/config.yml]: fixed in [d9c85ff](https://github.com/sepandhaghighi/pyrgg/pull/161/commits/d9c85fff6d627482a572ea7afcc5d2dd42377a84).

**Review by sepandhaghighi — APPROVED** [review]
LGTM 🚀

**Pull request merged**